Dockerfile

```diff
@@ -35,32 +69,39 @@ LABEL org.opencontainers.image.documentation="https://github.com/swissgrc/docker
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
 WORKDIR /
+
+# Install Git LFS
+
 # Copy Git LFS keyring
 COPY --from=build /etc/apt/keyrings/ /etc/apt/keyrings
 COPY --from=build /etc/apt/sources.list.d/ /etc/apt/sources.list.d
 
-# renovate: datasource=repology depName=debian_12/git versioning=loose
-ENV GIT_VERSION=1:2.39.2-1.1
-
-RUN apt-get update -y && \
-  # Install Git
-  apt-get install -y --no-install-recommends git=${GIT_VERSION} && \
-  # Clean up
-  apt-get clean && \
-  rm -rf /var/lib/apt/lists/* && \
-  # Smoke test
-  git version
-
-# Install Git LFS
-
 # renovate: datasource=github-tags depName=git-lfs/git-lfs extractVersion=^v(?<version>.*)$
 ENV GITLFS_VERSION=3.4.0
 
 RUN apt-get update -y && \
   # Install Git LFS
-  apt-get install -y --no-install-recommends git-lfs=${GITLFS_VERSION}  && \
-  # Clean up
-  apt-get clean && \
-  rm -rf /var/lib/apt/lists/* && \
+  # This will also install Git as dependency which will be overwritten later 
+  apt-get install -y --no-install-recommends git-lfs=${GITLFS_VERSION} && \
   # Smoke test
   git lfs version
+
+# Install Git 
+
+# renovate: datasource=repology depName=debian_12/libcurl3-gnutls versioning=loose
+ENV LIBCURL_VERSION=7.88.1-10+deb12u4
+
+# Install necessary dependencies
+RUN apt-get install -y --no-install-recommends libcurl3-gnutls=${LIBCURL_VERSION}
+
+# Copy Git
+COPY --from=build /usr/bin/git /usr/bin/git
```

Comment:
What is the best way to know whether a new versions of Git adds new folders to copy?

Reply:
Beside the individual files to copy, there are also other aspects which are relevant, like what packages are available for building (e.g. for HTTPS support). I therefore would suggest to later add a integration test to the pipeline, which runs the container and tries to clone a repo (which is the main use case for git inside the container).

Reply:
That would be great. Will you create an issue to track this?

Reply:
See #15